schemas/oval/5.11.3/independent-definitions-schema.xsd

```diff
@@ -2115,7 +2115,7 @@
                                                 </xsd:choice>
                                                 <xsd:element name="yamlpath" type="oval-def:EntityObjectStringType">
                                                       <xsd:annotation>
-                                                            <xsd:documentation>Specifies an YAML Path expression to evaluate against the YAML file specified by the filename entity. This YAML Path expression must evaluate to a list of zero or more scalar values which will be accessible in OVAL via instances of the value_of entity. Any results from evaluating the YAML Path expression other than a list of scalar values (e.g., a hash or list of lists) is considered an error. The intention is that the scalar values be drawn from instances of a single, uniquely named element. However, an OVAL interpreter is not required to verify this, so the author should define the YAML Path expression carefully. Note that "equals" is the only valid operator for the yamlpath entity.</xsd:documentation>
+                                                            <xsd:documentation>Specifies an YAML Path expression to evaluate against the YAML file specified by the filename entity. This YAML Path expression must evaluate to a list or a map (part of a map) of zero or more scalar values which will be accessible in OVAL via instances of the result entity. Any results from evaluating the YAML Path expression other than a list of scalar values (or a map with scalar values) at the bottom level (e.g. list of lists, list of maps, map with maps or lists) is considered an error. The intention is to drawn scalar values from instance(s) of a single (or multiple) flat element(s). However, an OVAL interpreter is not required to verify this, so the author should define the YAML Path expression carefully. Note that "equals" is the only valid operator for the yamlpath entity.</xsd:documentation>
```

Comment:
Description remarks:

- I believe that drawn -> draw
- What is a flat element?
- What is an OVAL interpreter not required to verify?

Reply:
Yes, "draw" is correct.

Flat element is probably a YAML key that contains only scalar. I think this needs to be clarified.

The OVAL interpreter isn't required to verify that the YAML path expresssion in the given object matches a scalar, a list of scalars or a map of scalars.

Reply:
Yeah, after reading it couple more times I realized that it needs some rephrasing.

Reply:
Fixed